containers/jekyll/.devcontainer/post-create.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+# If there's a Gemfile, then run `bundle install`
+# It's assumed that the Gemfile will install Jekyll too
+if [ -f Gemfile ]; then
```

Comment:
I  feel comfortable with the way this is implemented. Can you please add a description of this behavior to the documentation?